packages/editor/src/store/resolvers.js

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
`core-data` is meant to encompass all core entities, which in my mind should encompass autosaves. In other words, I think these resolvers and associated behavior ought to be moved there. Should help simplify things a fair bit as well (avoid reimplementing `SELECT`, `API_FETCH` controls).

Reply:
It looks like that would entail moving all the post state as well, since there's some coupling in the selectors:
https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/store/selectors.js#L493

Just want to confirm that's what you're proposing. Either I do that or use a different strategy - which I would imagine means moving the selector logic out into the components that use it.

Reply:
I managed to make this work by keeping `isEditedPostAutosaveable` in the editor store and passing autosave into it as an arg.

I still need to sort out deprecations for the old interface.